tests/Alert/AlertTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Widgets\Tests\Alert;
+
+use PHPUnit\Framework\TestCase;
+use Yiisoft\Definitions\Exception\CircularReferenceException;
+use Yiisoft\Definitions\Exception\InvalidConfigException;
+use Yiisoft\Definitions\Exception\NotInstantiableException;
+use Yiisoft\Factory\NotFoundException;
+use Yiisoft\Yii\Widgets\Alert;
+use Yiisoft\Yii\Widgets\Tests\Support\Assert;
+use Yiisoft\Yii\Widgets\Tests\Support\TestTrait;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
+ */
+final class AlertTest extends TestCase
+{
+    use TestTrait;
+
+    /**
+     * @throws CircularReferenceException|InvalidConfigException|NotFoundException|NotInstantiableException
```

Comment:
Maybe use separate `@throws` tag for each exception? That's how it usually is, and PHPDoc automatic generation works this way.

Reply:
Also maybe omit this it all for tests because they are internal?